R/vulnerability/disasters-emergencies/flooding/2022-interim/england/housing-characteristics/caravans.R

```diff
@@ -0,0 +1,25 @@
+library(readr)
+library(tidyr)
+library(dplyr)
+library(readxl)
+library(stringr)
+library(geographr)
+
+caci_demographics <- 
+  read_excel(
+    "C:/Users/040026704/British Red Cross Society/Strategic Insight and Foresight - Documents/Data/CACI/BRCS_LSOA_Up to Date Demographics_July_2021.xlsx",
+    skip = 10
+  )
+
+caravans <- 
+  caci_demographics %>% 
+  filter(str_detect(LSOA, "^E")) %>% 
+  select(lsoa_code = LSOA, caravans = `Caravan Or Other Mobile Or Temporary Structure`, households = `Total Households...39`) %>% 
```

Comment:
Not wrong at all as they are all the same number but just in case in future data they are different, think took denominator from 'Tenure' rather than 'Housing type':
```suggestion
  select(lsoa_code = LSOA, caravans = `Caravan Or Other Mobile Or Temporary Structure`, households = `Total Households...47`) %>%
```